spec/system/jobseekers_can_add_qualifications_to_their_job_application_spec.rb

```diff
@@ -11,66 +11,78 @@
   end
 
   context "adding a qualification" do
-    before do
-      visit jobseekers_job_application_build_path(job_application, :qualifications)
-      click_on I18n.t("buttons.add_qualification")
-    end
-
-    it "allows jobseekers to add a graduate degree" do
-      validates_step_complete(button: I18n.t("buttons.continue"))
-      select_qualification_category("Undergraduate degree")
-      expect(page).to have_content(I18n.t("jobseekers.job_applications.qualifications.new.heading.undergraduate"))
-      validates_step_complete(button: I18n.t("buttons.save_qualification.one"))
-      fill_in_undergraduate_degree
-      click_on I18n.t("buttons.save_qualification.one")
-      expect(current_path).to eq(jobseekers_job_application_build_path(job_application, :qualifications))
-      expect(page).to have_content(I18n.t("buttons.add_another_qualification"))
-      expect(page).to have_content("Undergraduate degree")
-      expect(page).to have_content("University of Life")
-      expect(page).not_to have_content("Subjects and grades")
-      expect(page).not_to have_content("School, college, or other organisation")
+    context "when JavaScript is not required" do
```

Comment:
i think required should be available?

Reply:
I separated the tests for journeys that require JS from journeys that don't require JS (these tests are for different journeys).

Reply:
id say that a journey a user is going on doesnt require or not require JS but the journey maybe different depending... thats why available makes more sense to me. really not a big deal tho, up to you

Reply:
The reason I want to separate them is that the journey for adding (eg) a degree doesn't need JS, which means I can test the radio buttons, but the journey for adding many secondary school qualifications does involve JS, which means I can't test the radio buttons